docker_compose/compose.yaml

```diff
@@ -0,0 +1,52 @@
+# To start, run:
+# % docker compose up
+#
+# What you should see:
+#   1. lots of waiting while containers are downloaded (or nothing if cached)
+#   2a. crawler should run and download 1 sample pdf
+#   2b. demo ui starting (in parallel with 2a)
+#   3. lots of logging as opensearch get started eventually 'Waiting for opensearch to exit.'
+#   4. sycamore container downloading pytorch files then running to import
+#   5. UI ready to go on port localhost:3000
+#
+# Troubleshooting/Extensions:
+#   1. Something is messed up, I just want to reset:
+#      Follow the instructions in reset.yaml
+#   2. I want more sort benchmark files:
```

Comment:
So, this docker compose is only going to be used for retrieving the sort benchmark data?  Can we allow users to provide their own url?

Reply:
I was going to figure out how to do additional things via compose after we got V1 out. Added to the V2 task list.

(You need to specify -a domain=... and -a url=... to the container run.